package.json

```diff
@@ -33,18 +33,20 @@
     "contrib/"
   ],
   "devDependencies": {
-    "@esotericsoftware/spine-core": "4.1.36",
     "@playcanvas/eslint-config": "^1.5.0",
+    "@rollup/plugin-alias": "^5.0.0",
+    "@rollup/plugin-commonjs": "^25.0.4",
+    "@rollup/plugin-node-resolve": "^15.2.1",
+    "@rollup/plugin-terser": "^0.4.3",
     "esbuild": "^0.19.0",
     "eslint": "^8.46.0",
-    "grunt": "^1.6.1",
-    "grunt-contrib-concat": "^2.1.0",
-    "grunt-contrib-uglify": "^5.2.2",
     "http-server": "^14.1.1",
-    "semver": "^7.5.4"
+    "rollup": "^3.28.1",
+    "semver": "^7.5.4",
+    "spine41": "npm:@esotericsoftware/spine-core@4.1"
   },
   "scripts": {
-    "build": "grunt",
+    "build": "rollup --config",
```

Comment:
Minor point but all our other `build` scripts use the `rollup -c` syntax.

Reply:
completed in: 9c7531d